iroh-p2p/src/node.rs

```diff
@@ -269,10 +272,12 @@ impl<KeyStorage: Storage> Node<KeyStorage> {
                     self.emit_network_event(NetworkEvent::PeerDisconnected(peer_id))
                         .await;
                 }
+
+                debug!(target: "dial", "ConnectionClosed: {:}", peer_id);
                 Ok(())
             }
             SwarmEvent::OutgoingConnectionError { peer_id, error } => {
-                debug!("failed to dial: {:?}, {:?}", peer_id, error);
+                debug!(target: "dial", "failed to dial: {:?}, {:?}", peer_id, error);
```

Comment:
why this `target: `addition?

Reply:
or rather, what does this do, we don't use this anywhere else

Reply:
ahh this is is how i've been filtering down to just the code path I want to view, cause I get so overwhelmed by the amount of output.

Adding a `target` means I can then do this:

`RUST_LOG=dial=debug iroh-ctl p2p connect` and just get logs that have target "dial"

I meant to remove it though because we don't use it anywhere else. Although, I will definitely vouch for its usefulness & can see it being very helpful for certain code paths.

You can also combo them, so say you have a few targets you want to follow at different log levels:
`RUST_LOG=dial=debug,other_target=trace`

Regardless, this one slipped by me, I need to remove it.